gaia-header.js

```diff
@@ -1,29 +1,46 @@
 ;(function(define){'use strict';define(function(require,exports,module){
+/*jshint laxbreak:true*/
 /*jshint esnext:true*/
+/*jshint boss:true*/
```

Comment:
in a project that uses gaia-header, I think the sane thing to do is to ignore the lib directory anyway :)